### ISSUE TITLE
We should document the k8s 1.24+ lack of auto-serviceaccounttoken generation

### ISSUE THREAD

**Issue by tico24** [crumbhole/argocd-vault-replacer#91]
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes

**Comment by tico24** [comment]
```
apiVersion: v1
kind: ServiceAccount
metadata:
  namespace: foo
  name: bar
secrets:
- name: fum
---
apiVersion: v1
kind: Secret
metadata:
  namespace: foo
  name: bar
  annotations:
    kubernetes.io/service-account.name: bar
type: kubernetes.io/service-account-token
```

**Comment by davidusken** [comment]
Made an attempt to document in this PR: https://github.com/crumbhole/argocd-vault-replacer/pull/92

**Comment by Joibel** [comment]
Closed by #92
